tests/hw-accel/neuron/README.md

```
@@ -135,10 +135,10 @@ $ export ECO_TEST_LABELS='neuron,vllm'
 $ export ECO_VERBOSE_LEVEL=100
 $ export ECO_HWACCEL_NEURON_DRIVERS_IMAGE="public.ecr.aws/q5p6u7h8/neuron-openshift/neuron-kernel-module:2.24.23.0"
 $ export ECO_HWACCEL_NEURON_DRIVER_VERSION="2.24.23.0"
-$ export ECO_HWACCEL_NEURON_DEVICE_PLUGIN_IMAGE="public.ecr.aws/neuron/neuron-device-plugin:2.24.23.0"
+$ export ECO_HWACCEL_NEURON_DEVICE_PLUGIN_IMAGE="public.ecr.aws/neuron/neuron-device-plugin:2.29.94.0"
```

markdownlint-cli2 (0.22.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)